src/modules/basic_auth/filter/basic_auth_filter.e

```diff
@@ -21,6 +21,8 @@ feature -- Basic operations
 		local
 			l_auth: HTTP_AUTHORIZATION
 		do
+				-- We can inherit from SHARED_LOGGER and write
```

Comment:
I would recommend as much as possible  to use api,logger. this is more flexible, especially in the future, we may want to put logs in the database for performance.